src/db/index.ts

```
@@ -0,0 +1,17 @@
+export const db = drizzle(sql, { schema })
+
+{/*export async function checkDatabaseConnection() {
+  try {
+    const result = await sql`SELECT 1`
+    console.log('Database connection successful')
+    return true
+  } catch (error) {
+    console.error('Database connection error:', error)
+    return false
+  }
+}*/}
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.

(lint/complexity/noUselessLoneBlockStatements)